target.h

```diff
@@ -115,7 +115,9 @@ class Target {
 	bool                 actionIdleExceeds(const ChronoDuration auto limit) const;
 	bool                 idleExceeds(const ChronoDuration auto limit) const;
 	void                 IncrementAction();
-	PositionVector       PredictedPositionAtTime(ChronoDuration auto interval);
+	PositionVector       PredictedPositionAtTime(ChronoDuration auto interval) {
```

Comment:
Please put this definition with the other inline functions, rather than in the class body.